src/Compilers/CSharp/Test/Semantic/Semantics/OverloadResolutionPerfTests.cs

```diff
@@ -551,5 +550,72 @@ public void NullableStateManyNestedFunctions()
                 //         return value;
                 Diagnostic(ErrorCode.WRN_NullReferenceReturn, "value").WithLocation(16395, 16));
         }
+
+        [WorkItem(51739, "https://github.com/dotnet/roslyn/issues/51739")]
+        [ConditionalFact(typeof(WindowsOnly), typeof(IsRelease))]
+        public void NullableAnalysisNestedExpressionsInMethod()
```

Comment:
Are these tests enough to induce the crash when we break the placement of Enter/ExitWalker?

Reply:
> Are these tests enough to induce the crash when we break the placement of Enter/ExitWalker?

Yes, they should be sufficient. The tests hit the crash without the change to `NullableWalker`.

---
In reply to: [594513255](https://github.com/dotnet/roslyn/pull/51851#discussion_r594513255) [](ancestors = 594513255)